internal/cli/config/get.go

```diff
@@ -53,27 +55,26 @@ func runGetCommand(cmd *cobra.Command, args []string) {
 		if err != nil {
 			feedback.Fatal(tr("Cannot get the key %[1]s: %[2]v", toGet, err), feedback.ErrGeneric)
 		}
-		feedback.PrintResult(getResult{key: toGet, data: resp.GetJsonData()})
+		var result getResult
+		err = json.Unmarshal([]byte(resp.GetJsonData()), &result.resp)
+		if err != nil {
+			feedback.Fatal(tr("Cannot parse JSON for key %[1]s: %[2]v", toGet, err), feedback.ErrGeneric)
+		}
+		feedback.PrintResult(result)
 	}
 }
 
 // output from this command may require special formatting.
 // create a dedicated feedback.Result implementation to safely handle
 // any changes to the configuration.Settings struct.
 type getResult struct {
-	key  string
-	data interface{}
+	resp interface{}
 }
 
 func (gr getResult) Data() interface{} {
-	return gr.data
+	return gr.resp
 }
 
 func (gr getResult) String() string {
-	gs, ok := gr.data.(string)
-	if !ok {
-		// Should never happen
-		panic(tr("Cannot get key %s value as string: %v", gr.key, gr.data))
-	}
-	return gs
+	return fmt.Sprintf("%v", gr.resp)
```

Comment:
Printing the result this way is not optimal. When I try to get configs using a key that returns a map, this is the output:
```
$ ./arduino-cli.exe config get metrics
map[addr::9090 enabled:true]
```
It is, of course, correct, but we should display it more clearly. I think that changing this `String()` function to match the one from [config dump](https://github.com/arduino/arduino-cli/blob/master/internal/cli/config/dump.go#L55-L62). Trying it locally, this is the output I get:
```
$ ./arduino-cli.exe config get metrics
addr: :9090
enabled: true

$ ./arduino-cli.exe config get metrics.addr
:9090
```

Reply:
I was expecting the user to provide `--format` flag if they wished to have a particular format:

```
$ arduino-cli config get metrics --format yaml
addr: :9090
enabled: true

$ arduino-cli config get metrics.addr --format yaml
:9090
```

Otherwise, this `text`  format is impossible to generate, which I find useful as an unambiguous syntax reference.

On the other hand, the subcommands of `config` (`dump`, `get`) should probably generate the same format by default, and people probably already depend on the format of `dump`.

So I'll concede and accept the change :)